scripts/download_allsets.js

```diff
@@ -67,22 +67,18 @@ const fetchZip = () => (
   }));
 
 const download = async () => {
-  try {
-    logger.info("Checking if AllSets.json is up to date");
-    const [isUpToDate, version] = await isVersionUpToDate();
-    if (!isUpToDate) {
-      await fetchZip();
-      logger.info("Fetch AllSets.json finished. Updating the cards and sets data");
-      updateDatabase();
-      logger.info("Update DB finished");
-      fs.writeFileSync(setsVersion, version);
-      refreshVersion();
-      await downloadBoosterRules();
-    } else {
-      logger.info("AllSets.json is up to date");
-    }
-  } catch(err) {
-    logger.error(`Couldn't complete AllSets.json check : ${err}`);
```

Comment:
This change seems unrelated to this feature?

(let me know if new people reviewing parts of a PR is not helpful)

Reply:
It is very helpful, also well-spotted!

This change is taken from a patch that I had to merge into my feature branch during development and it got squashed in.

It's already upstream, so it won't cause any trouble.